src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -647,7 +647,7 @@ def download_file(self, file_name: str, local_directory: str | None = None):
             else:
                 raise FileNotFoundError("Remote file does not exist.")
 
-    def download(self, file_name: list[str] | str, local_directory: str | None = None):
+    def download(self, file_name: list[str] | str, local_directory: str | None = "."):
```

Comment:
@MohammedAnsys The changes in this file are not relevant to this PR. These are old changes. We should resolve this before merging the PR.

Reply:
@MohammedAnsys This is resolved properly. Unresolving again.

Reply:
It's resolved now.